client/interface_peer.go

```diff
@@ -0,0 +1,96 @@
+package client
+
+import (
+	"github.com/go-routeros/routeros"
+)
+
+type InterfacePeer struct {
+	Id                     string `mikrotik:".id"`
+	AllowedAddress         string `mikrotik:"allowed-address"`
+	Comment                string `mikrotik:"comment"`
+	Disabled               bool   `mikrotik:"disabled"`
+	EndpointAddress        string `mikrotik:"endpoint-address"`
+	EndpointPort           int    `mikrotik:"endpoint-port"`
+	Interface              string `mikrotik:"interface"`
+	PersistentKeepalive    int    `mikrotik:"persistent-keepalive"`
+	PresharedKey           string `mikrotik:"preshared-key"`
+	PublicKey              string `mikrotik:"public-key"`
+	CurrentEndpointAddress string `mikrotik:"current-endpoint-address,readonly"`
+	CurrentEndpointPort    string `mikrotik:"current-endpoint-port,readonly"`
+	LastHandshake          string `mikrotik:"last-handshake,readonly"`
+	Rx                     string `mikrotik:"rx,readonly"`
+	Tx                     string `mikrotik:"tx,readonly"`
+}
```

Comment:
All of these are integer types as specified in [the docs.](https://help.mikrotik.com/docs/display/ROS/WireGuard#WireGuard-Peerconfiguration) We should switch these to `int` types rather than strings.

![Screenshot_20230531_235905](https://github.com/ddelnano/terraform-provider-mikrotik/assets/5855593/e3be8a82-9e7c-4a4d-9914-7ec58a21d398)

Also with respect to these fields and the upcoming `mikrotik_interface_wireguard_peer` resource changes, please omit `LastHandshake`, `Rx` and `Tx` from the terraform resource. These values will change between terraform runs and so it will create a frustrating experience when each subsequent `terraform apply` causes changes to occur.

If that doesn't make sense, we can talk more about it in Slack and I can show you what I mean.